Pipfile

```diff
@@ -11,9 +10,15 @@ numpy = "*"
 matplotlib = "*"
 pandas = "*"
 jupyter = "*"
-jinja = "*"
 sklearn = "*"
 jinja2 = "*"
+flake8 = "*"
```

Comment:
always choose a specific version. Otherwise you can (in the future) break your software

Reply:
Oh, forgot about it. I will fix it soon.